org.eclipse.lsp4j.debug/src/main/java/org/eclipse/lsp4j/debug/DebugProtocol.java

```diff
@@ -0,0 +1,11631 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Kichwa Coders Ltd. and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+
+package org.eclipse.lsp4j.debug;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Objects;
+
+import org.eclipse.lsp4j.jsonrpc.messages.Either;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * <p>
+ * Declaration of parameters, response bodies, and event bodies.
+ * </p>
+ * <p>
+ * Auto-generated from debugProtocol.json schema version 1.24.0. Do not edit
+ * manually.
+ * </p>
+ */
+public class DebugProtocol {
+	/**
+	 * <p>
+	 * Version of debugProtocol.json this class was derived from.
+	 * </p>
+	 */
+	public static final String SCHEMA_VERSION = "1.24.0";
+
+	/**
+	 * <p>
+	 * Event message for 'stopped' event type.
+	 * </p>
+	 * <p>
+	 * The event indicates that the execution of the debuggee has stopped due to
+	 * some condition.
+	 * </p>
+	 * <p>
+	 * This can be caused by a break point previously set, a stepping action has
+	 * completed, by executing a debugger statement etc.
+	 * </p>
+	 */
+	public static class StoppedEventArguments {
+
+		/**
+		 * <p>
+		 * The reason for the event.
+		 * </p>
+		 * <p>
+		 * For backward compatibility this string is shown in the UI if the
+		 * 'description' attribute is missing (but it must not be translated).
+		 * </p>
+		 * <p>
+		 * Possible values include - but not limited to those defined in {@link Reason}
+		 * </p>
+		 */
+		public interface Reason {
+			/**
+			 * <p>
+			 * Encoded value: {@code step}
+			 * </p>
+			 */
+			public static final String STEP = "step";
+			/**
+			 * <p>
+			 * Encoded value: {@code breakpoint}
+			 * </p>
+			 */
+			public static final String BREAKPOINT = "breakpoint";
+			/**
+			 * <p>
+			 * Encoded value: {@code exception}
+			 * </p>
+			 */
+			public static final String EXCEPTION = "exception";
+			/**
+			 * <p>
+			 * Encoded value: {@code pause}
+			 * </p>
+			 */
+			public static final String PAUSE = "pause";
+			/**
+			 * <p>
+			 * Encoded value: {@code entry}
+			 * </p>
+			 */
+			public static final String ENTRY = "entry";
+		}
+
+		/**
+		 * <p>
+		 * The reason for the event.
+		 * </p>
+		 * <p>
+		 * For backward compatibility this string is shown in the UI if the
+		 * 'description' attribute is missing (but it must not be translated).
+		 * </p>
+		 * <p>
+		 * Possible values include - but not limited to those defined in {@link Reason}
+		 * </p>
+		 */
+		public String reason;
```

Comment:
The fields should not be public, but accessible via getters, in order to be consistent with the LSP implementation.